Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10875,10 +10883,39 @@ def xsiam_api_call_with_retries(
         if status_code == 429:
             time.sleep(1)
         attempt_num += 1
-
+    if is_json_response:
+        response = response.json()
+    if response.get('error', '').lower() != 'false':
+        raise DemistoException(header_msg + response.get('error'))
```

Comment:
The error check can only be done for json, so this check should be done only for that case.
```suggestion
        if response.get('error', '').lower() != 'false':
            raise DemistoException(header_msg + response.get('error'))
```